backend/routes/post.py

```
@@ -185,6 +187,22 @@ async def create_text_post(
         raise HTTPException(status_code=400, detail="Inappropriate content detected")
 
     post = create_post_entry(db, current_user.id, content, "text")
+    hashtags = extract_hashtags(post.content)
```

Ruff (0.8.2)

`extract_hashtags` may be undefined, or defined from star imports

(F405)